### ISSUE TITLE
Upload image permission error

### ISSUE THREAD

**Issue by dannyalfonzo** [tzyganu/UMC1.9#114]
Hi guys, I am Using permissions 644 for files and 755 for directories 

with magento 1.9.1.0 when uploading a picture, it is with its perfectly read permissions

But with 1.9.2.3 when uploading a picture, It is not with read permissions

# ls -l | grep kia_2.png
-rw-r--r--. 1 digivg digivg 28206 Feb 15 18:41 kia_2.png

what can I do?

PD: sorry for my bad english

**Comment by tzyganu** [comment]
then this is not a UMC issue. It may be a magento issue. Can you upload images in other modules? like for products?

**Comment by dannyalfonzo** [comment]
Yes, the issue is form magento http://stackoverflow.com/questions/35024735/magento-1-9-2-3-after-uploading-images-from-backend-permission-is-set-to-640-i

Thanks
